js/src/modal.js

```diff
@@ -452,7 +452,7 @@ class Modal {
 
   _checkScrollbar() {
     const rect = document.body.getBoundingClientRect()
-    this._isBodyOverflowing = rect.left + rect.right < window.innerWidth
+    this._isBodyOverflowing = Math.round(rect.left + rect.right) < window.innerWidth
```

Comment:
I was wondering, can't this result in equal values rounded up? If so, shouldn't the check be adapted?

Reply:
Maybe using `parseInt` would be better here, for example `Math.round(5.5)` will give you `6` but `parseInt(5.5)` will give you `5` 🤔

Reply:
Or https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor ?